src/app/records/PresentationNodeLeaf.tsx

```diff
@@ -13,17 +15,103 @@ export default function PresentationNodeLeaf({
   ownedItemHashes,
   completedRecordsHidden,
   redactedRecordsRevealed,
+  sortRecordProgression,
 }: {
   node: DimPresentationNodeLeaf;
   ownedItemHashes?: Set<number>;
   completedRecordsHidden: boolean;
   redactedRecordsRevealed: boolean;
+  sortRecordProgression: boolean;
 }) {
+  const newNode = { ...node };
+  if (sortRecordProgression) {
+    /**
+     * RECORDS
+     */
+    if (node.records) {
+      newNode.records = orderBy(
+        newNode.records,
+        (record) => {
+          /**
```

Comment:
This `/**` is a JSDoc comment appropriate for documenting things that should show up in intellisense. For internal comments on code, use line comments `//`.